### ISSUE TITLE
accessibility of mass unlocking tips is nonexistent on the lbry mac app using voiceover

### ISSUE THREAD

**Issue by marrie** [lbryio/lbry-desktop#4044]
The accessibility of the mass unlock of tipping in the app does not work as of yet. I'm using voiceover on the mac with the app. I reported this already  I believe in terms of the lbry.tv website but here are  the  steps.

1. Launch voiceover. This can be done with command f5.
2. Launch the lbry app.
3 turn off your screen.
4. Navigate to the wallet link.
5. Pick a video you published. Note that the ability to see tips is gone for individual unlocking. At least I cannot find it.
6. Find a way to move the slider if there is one. This is where the problem is. I cannot interact with the table, I mean I can, but the slider is invisible to voiceover, or appears to be.

**Comment by seanyesmunt** [comment]
I'm really sorry about this @marrie 

Thank you for creating this issue. I assigned it to myself and I will start working on improving this.

**Comment by marrie** [comment]
Thanks so much. I hope those steps were clear enough. I kind of wrote 
those in a hurry. I did not test the windows client, but I assume that 
we might have the same issues as well. Can that be tested with 
[nvda?](http://nvda-project.org) Please? Thanks for the quick comment.

**Comment by marrie** [comment]
# Another comment:
I tested this on windows and I don't really want to create another bug 
for this as the steps are going to be the same. Mass unlocking tips is 
not doable in the app. I do have the work around, but this is to let you 
guys know anyway that the current function of mass unlocking is not 
accessible for any screen reader as far as I can tell. The slider 
control is just not showing up.

Thanks so much.

**Comment by seanyesmunt** [comment]
This should be fixed in next version

**Comment by marrie** [comment]
Yep. It almost is accessible. The only issue now is the button is 
mislabeled on mac os, I have not yet tried this on windows yet. But from 
my testing, it  is 99 percent of the way there. Thanks so much.

**Comment by seanyesmunt** [comment]
@marrie Which button is mislabled? I'll fix that right now.

**Comment by marrie** [comment]
Thanks, when you go to the tips section on a video, expand it, then 
curser through the table to the tips seciton, there is a link and a 
button that says somethingn like object button. You will have to use 
voiceover to see the label. I have not tested this with NVDA as of yet. 
Here's hoping though that will fix both nvda and voiceover labeling 
issues. I can't code, I just brake, I mean test things. Lol!

Thanks, let me know if you have questions.
